examples/step-63/step-63.cc

```diff
@@ -1004,34 +1032,38 @@ namespace Step63
   }
 
 
-  // @sect4{<code>AdvectionProblem::solve</code>}
+  // @sect4{<code>AdvectionProblem::solve()</code>}
 
   // Before we can solve the system, we must first set up the multigrid
   // preconditioner. This requires the setup of the transfer between levels,
   // the coarse matrix solver, and the smoother. This setup follows almost
   // identically to Step-16, the main difference being the various smoothers
   // defined above and the fact that we need different interface edge matrices
-  // for in and out since our problem is non-symetric. (In reality, for this
+  // for in and out since our problem is non-symmetric. (In reality, for this
   // tutorial these interface matrices are empty since we are only using global
   // refinement, and thus have no refinement edges. However, we have still
   // included both here since if one made the simple switch to an adaptively
   // refined method, the program would still run correctly.)
 
-  // The last thing to note is that since our problem is non-symetric, we must
+  // The last thing to note is that since our problem is non-symmetric, we must
   // use an appropriate Krylov subspace method. We choose here to
   // use GMRES since it offers the guarantee of residual reduction in each
-  // iteration. The major disatvantage to GMRES is that, for each iteration, we
-  // must store an additional temporary vector as well as compute an additional
-  // scalar product. This requirement is relaxed by using the restarted GMRES
+  // iteration. The major disavantage of GMRES is that, for each iteration, we
+  // the number of stored temporary vectors increases by one, and one also needs
+  // to compute a scalar product with all previously stored vectors. This is
+  // rather expensive. This requirement is relaxed by using the restarted GMRES
   // method which puts a cap on the number of vectors we are required to store
-  // at any one time (here we resart after 50 temporary vectors, or 48
+  // at any one time (here we restart after 50 temporary vectors, or 48
   // iterations). This then has the disatvantage that we lose information we
```

Comment:
```suggestion
  // iterations). This then has the disadvantage that we lose information we
```

Reply:
Ah, darn. I saw that and thought I had fixed it :-(